ui/src/styles/index.scss

```diff
@@ -141,6 +141,12 @@
         content: "*";
       }
     }
+    .ant-form-item-control {
+      .ant-form-item-extra {
```

Comment:
Are these overrides mandated by the designs?

Reply:
The designs are not yet updated with these info messages, [this was recently agreed upon on Slack](https://0xpolygon.slack.com/archives/C03HUC0Q7RU/p1681370779487889). I added these overrides because the default looks ugly IMO:

![input-extra-position](https://user-images.githubusercontent.com/15896177/232723180-0cf4d233-d676-4899-9a01-0d212bd113be.gif)